addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -343,6 +345,10 @@ export const dateTimeField = {
             ),
         },
     ],
+    extractProps: ({ attrs, options }, dynamicInfo) => ({
```

Comment:
you should declare the option in the supported options above (for studio), like we do here <https://github.com/odoo/odoo/pull/162373/files#diff-a9e557504c093b4816b3a5eb384daad3c171eb4f40442fe20eee084df423446bR343>